test/Tests.Infrastructure/RavenRetryTheoryAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using xRetry;
+
+namespace Tests.Infrastructure
+{
+    public class RavenRetryTheoryAttribute : RetryTheoryAttribute
```

Comment:
Instead of adding a new attribute can we make `RavenTheoryAttribute` to inherit from `RetryTheoryAttribute` and by default set max retries to 0? Something like:

```
public class RavenTheoryAttribute : RetryTheoryAttribute, ITraitAttribute
{

    public RavenTheoryAttribute(RavenTestCategory category) : base(maxRetries: 0)
    {
    }

    public RavenTheoryAttribute(RavenTestCategory category, int maxRetries, int delayBetweenRetriesMs = 0, params Type[] skipOnExceptions)
        : base(maxRetries, delayBetweenRetriesMs, skipOnExceptions)
    {
    }
```

Then please apply the same to `RavenFactAttribute` so we'll have the retry option there as well